.gitignore

```diff
@@ -357,3 +357,8 @@ config.json
 
```

Comment:
What is this file doing exactly?

Reply:
gitignore file and folder generated by recompiling feast-azure-provider